integrationtesting/opendcs-tests/src/test/java/org/opendcs/spi/configuration/Configuration.java

```diff
@@ -109,4 +114,12 @@ default public boolean supportsDao(Class<? extends DaoBase> dao)
     * @return
     */
     public String getName();
+
+    void storeScheduleEntryStatus(ScheduleEntryStatus status) throws DatabaseException;
```

Comment:
These methods don't belong in this interface, you can get access to all these DAO's from the REST API already